src/components/form/form.style.js

```diff
@@ -0,0 +1,101 @@
+import styled, { css, keyframes } from 'styled-components';
+import StyledFormField from '../../__experimental__/components/form-field/form-field.style';
+import StyledButton from '../button/button.style';
+import { isClassic } from '../../utils/helpers/style-helper';
+
+export const StyledAdditionalFormAction = styled.div`
+  ${({ type }) => type && css`
+    ${type === 'leftAlignedActions' && css`
+      flex-grow: 1;
+    `}
+
+    ${type !== 'leftAlignedActions' && css`
+      padding-left: 15px;
+      display: inline-block;
+    `}
+
+    ${StyledButton} {
+      &:first-child {
+        margin-left: 0;
+      }
+    }
+  `}
+`;
+
+const FormButtonAnimation = keyframes`
+  0%   { bottom: -50px; }
+  100% { bottom: 0; }
+`;
+
+export const StyledFormFooter = styled.div`
+  ${({ theme, stickyFooter }) => !isClassic(theme) && !stickyFooter && css`
+    margin-top: 48px;
+  `}
+
+  ${({ buttonAlign }) => buttonAlign === 'right' && css`
+    .carbon-app-wrapper{
+      justify-content: flex-end;
+    }
+  `}
+`;
+
+const StyledForm = styled.form`
+  && ${StyledButton} {
+    margin-left: 16px;
+    ${({ theme }) => !isClassic(theme) && css`margin-left: 15px;`}
+  }
+
+  ${({ theme }) => !isClassic(theme) && css`
```

Comment:
there is the `isClassic` helper to check for the `classic` theme

Reply:
Yeah that's what I'm using